mountpoint-s3/examples/prefetch_benchmark.rs

```diff
@@ -78,28 +92,38 @@ fn main() {
     }
     let client = Arc::new(S3CrtClient::new(config).expect("couldn't create client"));
 
+    let head_object_result = block_on(client.head_object(bucket, key)).expect("HeadObject failed");
+    let size = head_object_result.object.size;
+    let etag = ETag::from_str(&head_object_result.object.etag).unwrap();
+
     for i in 0..iterations.unwrap_or(1) {
-        let runtime = ThreadPool::builder().pool_size(1).create().unwrap();
+        let runtime = client.event_loop_group();
```

Comment:
Why do we move this from a separate thread pool onto the CRT's event loop? This doesn't match what we do with Mountpoint.

Reply:
My mistake, I was thinking we were using it for the prefetch reads but we're passing this into the prefetcher _which does match what we're doing with MP_.

Reply:
In fact, this is what we do in Mountpoint: we get the [event loop from the client](https://github.com/awslabs/mountpoint-s3/blob/ed4735d71af0432491a361912da747ccaf39a21b/mountpoint-s3/src/cli.rs#L693) and use it to [initialize the prefetcher](https://github.com/awslabs/mountpoint-s3/blob/ed4735d71af0432491a361912da747ccaf39a21b/mountpoint-s3/src/cli.rs#L812).